example/cloudexec.toml

```diff
@@ -4,11 +4,66 @@ timeout = "48h"
 
 [commands]
 setup = '''
+
+########################################
```

Comment:
We should also put this boilerplate setup in the launch init https://github.com/crytic/cloudexec/blob/83ef5d89f99ef9f595a8aae54f4dbb6013732b9a/cmd/cloudexec/launch.go#L55

Reply:
disregard this if moving critical setup to user_data